jsonresolver/core.py

```diff
@@ -45,5 +45,5 @@ def resolve(self, url):
         if self.url_map is None:
             self._build_url_map()
         parts = urlsplit(url)
-        loader, args = self.url_map.bind(parts.hostname).match(parts.path)
+        loader, args = self.url_map.bind(parts.netloc).match(parts.path)
```

Comment:
After host matching was enabled, we could only match ``localhost`` and not e.g. ``localhost:4000``. Hence ``@jsonresolver.route('...', host='localhost:4000')``, which was used many places in the tests would not work.
This change fixes that, but I'm not sure if it's desirable to match against port number as well.